### DISCUSSION TITLE
If I compile a Fyne app for IOS or Android, do I have access to the entire Go library?

### DISCUSSION THREAD

**Discussion by serge-hulne** [fyne-io/fyne#3601] [General]
If I compile a Fyne app for IOS or Android, do I have access to the entire Go library or is it a "sandboxed" or partial implementation/version of the Go standard library?

**Comment by andydotxyz** [comment]
It is the complete Go library, not a cut down version.

However we do recommend using the Fyne `storage` API over the standard file access because otherwise you can suffer from the OS sandbox restrictions which we have worked to overcome.
